main.go

```diff
@@ -80,11 +97,11 @@ func main() {
 	// Healthcheck API
 	versionInfo, err := healthcheck.NewVersionInfo(BuildTime, GitCommit, Version)
 	if err != nil {
-		log.Fatal(ctx, "Failed to obtain VersionInfo for healthcheck", err)
+	log.Fatal(ctx, "Failed to obtain VersionInfo for healthcheck", err)
 	}
 	hc := healthcheck.New(versionInfo, cfg.HealthcheckCriticalTimeout, cfg.HealthcheckInterval)
 	if err = hc.AddCheck("API router", zebedeeClient.Checker); err != nil {
-		log.Fatal(ctx, "Failed to add api router checker to healthcheck", err)
+	log.Fatal(ctx, "Failed to add api router checker to healthcheck", err)
```

Comment:
Could the indentation be replaced please as this will likely fail linting checks otherwise.